Maddori.Apple/Maddori.Apple/Screen/Home/HomeViewController.swift

```diff
@@ -37,12 +98,56 @@ final class HomeViewController: BaseViewController {
     }
     
     override func render() {
+        view.addSubview(teamNameLabel)
+        teamNameLabel.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide).offset(SizeLiteral.topPadding)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        view.addSubview(invitationCodeButton)
+        invitationCodeButton.snp.makeConstraints {
+            $0.leading.equalTo(teamNameLabel.snp.trailing).offset(Size.labelButtonPadding)
+            $0.width.equalTo(Size.subButtonWidth)
+            $0.height.equalTo(Size.subButtonHeight)
+            $0.bottom.equalTo(teamNameLabel.snp.bottom).offset(-5)
+            // offset을 없애면 너무 낮은 것 같다는 생각에 임의로 줘봤습니다
+        }
+        
+        view.addSubview(descriptionLabel)
+        descriptionLabel.snp.makeConstraints {
+            $0.top.equalTo(teamNameLabel.snp.bottom).offset(Size.labelPadding)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        view.addSubview(currentReflectionLabel)
+        currentReflectionLabel.snp.makeConstraints {
+            $0.top.equalTo(descriptionLabel.snp.bottom).offset(Size.propertyPadding)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        view.addSubview(addFeedbackButton)
+        addFeedbackButton.snp.makeConstraints {
+            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+            $0.bottom.equalTo(view.safeAreaLayoutGuide).offset(SizeLiteral.bottomPadding)
+            $0.height.equalTo(Size.mainButtonHeight)
+        }
+        
+        view.addSubview(planLabelButtonView)
+        planLabelButtonView.snp.makeConstraints {
+            $0.centerX.equalToSuperview()
+            $0.bottom.equalTo(addFeedbackButton.snp.top)
+            $0.height.equalTo(SizeLiteral.minimumTouchArea)
+        }
+        
         view.addSubview(keywordCollectionView)
         keywordCollectionView.snp.makeConstraints {
-            $0.edges.equalToSuperview()
+            $0.top.equalTo(currentReflectionLabel.snp.bottom).offset(Size.labelPadding)
+            $0.leading.equalTo(view.safeAreaLayoutGuide)
+            $0.trailing.equalTo(view.safeAreaLayoutGuide)
```

Comment:
p4;

요 두개는 
```suggestion
            $0.leading.trailing.equalTo(view.safeAreaLayoutGuide)
```
하나로 합칠 수 있습니다 !

Reply:
[f2ce361](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/36/commits/f2ce3613ecef61dce0d87e9d5415dd86eefd7b2a)
ㅎ.. .그러게요 왜 안합쳤지